ecosystem/sep-0005.md

```diff
@@ -96,6 +114,25 @@ m/44'/148'/8' GDJTCF62UUYSAFAVIXHPRBR4AUZV6NYJR75INVDXLLRZLZQ62S44443R SCD5OSHUU
 m/44'/148'/9' GBTVYYDIYWGUQUTKX6ZMLGSZGMTESJYJKJWAATGZGITA25ZB6T5REF44 SCJGVMJ66WAUHQHNLMWDFGY2E72QKSI3XGSBYV6BANDFUFE7VY4XNXXR
 ```
 
+Shamir Secret Sharing shares (non-deterministic) - group threshold = 2; groups = [(2, 3), (1, 1), (2, 3)]:
```

Comment:
The Stellar keys of BIP 39 and the Stellar keys of SLIP 39 are different. Are you willing to add it?

Additionally, this part only applies to BIP-39. Can you specify that? For example, `m/44'/148'` key (BIP39):
> `m/44'/148'` key:
> ```
> e0eec84fe165cd427cb7bc9b6cfdef0555aa1cb6f9043ff1fe986c3c8ddd22e3
> ```

Reply:
> The Stellar keys of BIP 39 and the Stellar keys of SLIP 39 are different. Are you willing to add it?

Sorry I don't understand what you mean.

Reply:
Oh I see now (linked to the other PR.) If I start from the entropy and use it directly then yes keys will be different. To get the same keys I need to go with the `to_bip39` as I had before. I wanted to have everything being consistent with a single set of Stellar Keys. But if we can't because we say we don't want to use BIP39 as transition then I can make a specific section for SLIP39.